homeassistant/components/mqtt/client.py

```diff
@@ -547,11 +547,11 @@ def async_remove() -> None:
                 raise HomeAssistantError("Can't remove subscription twice")
             self.subscriptions.remove(subscription)
             if wildcard_topic(topic):
```

Comment:
There is a conflict with a recent PR that needs to be sorted out.  It should hopefully make this a bit easier to manage this block of code as well.